spec.bs

```diff
@@ -1635,11 +1636,12 @@ An interest group is a [=struct=] with the following items:
 : <dfn>expiry</dfn>
 :: A point in time at which the browser will forget about this interest group.
 : <dfn>owner</dfn>
-:: An [=origin=]. Frames that join interest groups owned by `owner` must either be served from origin
-  `owner`, or another origin delegated by `owner` (See [=checking interest group permissions=] for details).
-  The [=origin/scheme=] must be "<code>https</code>".
+:: An [=origin=]. Frames that join interest groups owned by `owner` must either be served from
+  origin `owner`, or another origin delegated by `owner` (See [=checking interest group permissions=]
+  for details). The [=origin/scheme=] must be "`https`".
 : <dfn>name</dfn>
-:: A [=string=]. The ([=interest group/owner=], [=interest group/name=]) tuple is a key that uniquely defines each interest group.
+:: A [=string=]. The ([=interest group/owner=], [=interest group/name=]) tuple is a key that
+  uniquely defines each interest group.
 : <dfn>priority</dfn>
 :: A {{double}}. Defaulting to 0.0. Used to select which interest groups participate in an auction

```

Comment:
I think we need a TODO here to link to a section that describes this, ditto for the related fields.